Makefile

```diff
@@ -21,7 +21,7 @@ ifeq ($(SYSTEM),Darwin)
 	SHARED = -dynamiclib
 	SHAREDSUFFIX = .dylib
 	CC = /usr/bin/gcc
-	CPPFLAGS += -I/opt/local/include -L/opt/local/lib
+	CPPFLAGS += -I/user/local/include -L/usr/local/lib
```

Comment:
`/usr/local/include`?

Reply:
yes it is where all the headers for the libraries are installed using `brew`  are

```
ls /usr/local/include/
SDL	  gdbm.h       idn2.h	libpng16    lzma    pngconf.h	  snappy-sinksource.h	 ucl	    unictype.h	uniname.h   unistr.h	uniwbrk.h   zip.h
SDL2	  gpg-error.h  libelf	libusb-1.0  lzma.h  pnglibconf.h  snappy-stubs-public.h  unicase.h  unigbrk.h	uninorm.h   unistring	uniwidth.h  zipconf.h
gcrypt.h  gpgrt.h      libltdl	ltdl.h	    png.h   snappy-c.h	  snappy.h		 uniconv.h  unilbrk.h	unistdio.h  unitypes.h	usb.h
```

Reply:
Thanks @uyjlian!
Was a typo issue, it is fixed now